src/battle_script_commands.c

```diff
@@ -1914,14 +1914,13 @@ s32 CalcCritChanceStageArgs(u32 battlerAtk, u32 battlerDef, u32 move, bool32 rec
     s32 critChance = 0;
 
     if (gSideStatuses[battlerDef] & SIDE_STATUS_LUCKY_CHANT
-        || gStatuses3[battlerAtk] & STATUS3_CANT_SCORE_A_CRIT)
```

Comment:
Any reason you removed STATUS3_CANT_SCORE_A_CRIT? Could be useful if someone implemented a move which blocks critical hits.

Reply:
Not real reason other then it isn't used anywhere else. I can add it back.

Reply:
I guess you can add it back for now, and later we'll decide what to do with that

Reply:
added it back